configure.ac

```diff
@@ -58,6 +58,9 @@ dnl
 PICFLAGS="$lt_prog_compiler_pic"
 AC_SUBST([PICFLAGS])
 
+AC_LIBTOOL_COMPILER_OPTION([if $compiler supports -fno-math-errno], [_cv_nomatherrno], [-fno-math-errno], [], [CFLAGS="$CFLAGS -fno-math-errno"], [])
```

Comment:
Do these flags provide significant performance gain? I get some discomfort from venturing outside of IEEE floating point.